packages/core/src/http/requestBuilder.ts

```diff
@@ -91,6 +91,13 @@ export type ApiErrorConstructor = new (
   message: string
 ) => any;
 
+export type ErrorType<ErrorCtorArgs extends any[]> = {
```

Comment:
Usually, you want to use interface types when you're describing an object interface that gets referenced/reused a lot.
```suggestion
export interface ErrorType<ErrorCtorArgs extends any[]>{
```